frameit/lib/frameit/screenshot.rb

```diff
@@ -41,7 +41,7 @@ def device_name
         return 'iPhone 4'
       when sizes::IOS_IPAD
         return 'iPad Air 2'
-      when sizes::IOS_IPAD_PRO
+      when sizes::IOS_IPAD_PRO, sizes::IOS_IPAD_10_5
```

Comment:
https://github.com/fastlane/fastlane/pull/12837/files#diff-31f55c7b2572839daddfa36385dfaa44R13 says `attr_accessor :screen_size # deliver screen size type, is unique per device type, used in device_name`

Does this need to be its own use case and return something like `"iPad Pro 10.5"`?